mobile/views/TabBar/DirectMessageStack.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+import * as React from 'react';
+import { Button } from 'react-native';
+import { StackNavigator } from 'react-navigation';
+import idx from 'idx';
```

Comment:
- Added `idx` to the deps and used it throughout, shoutout to @alejandronanez for the tip

Reply:
Awesome!